src/Coordination/KeeperServer.cpp

```diff
@@ -316,6 +316,21 @@ void KeeperServer::launchRaftServer(const Poco::Util::AbstractConfiguration & co
         }
     }
 
+    params.leadership_expiry_ = getValueOrMaxInt32AndLogWarning(
+        coordination_settings->leadership_expiry_ms.totalMilliseconds(), "leadership_expiry_ms", log);
+
+    if (params.leadership_expiry_ > 0)
+    {
+        if (params.leadership_expiry_ < params.election_timeout_lower_bound_)
+        {
+            LOG_WARNING(
```

Comment:
if it can be used to avoid multiple leaders then this makes more sense as `LOG_INFO`.